src/pages/docs/infrastructure/deployment-targets/tentacle/windows/index.mdx

```diff
@@ -1,85 +1,96 @@
+
+If your network rules only allow port **80** and **443** to the Octopus Server, you can either:
+- Change the server bindings to either HTTP or HTTPS and use the remaining port for polling Tentacle connections.
+  - The listening port Octopus Server uses can be [changed from the command line](/docs/octopus-rest-api/octopus.server.exe-command-line/configure) using the `--commsListenPort` option.
+Even if you do use port **80** for Polling Tentacles, the communication is still secure.
+- Use a reverse proxy to redirect incoming connections to the Tentacle listening port on Octopus Server by differentiating the connection based on Hostname (TLS SNI) or IP Address
+  - The polling endpoint Tentacle uses can be [changed from the command line](/docs/infrastructure/deployment-targets/tentacle/polling-tentacles-over-port-443/#self-hosted) using the `--server-comms-address` option. 
+  - You can learn about this configuration on the [Polling Tentacles over standard HTTPS port](/docs/infrastructure/deployment-targets/tentacle/polling-tentacles-over-port-443) page.
+
+Note that the port (or address) used to poll Octopus for jobs is different from the port (or address) used by your team to access the Octopus Deploy web interface;
+this is on purpose, and it means you can use different firewall conditions to allow Tentacles to access the Octopus Server by IP address.
+
+Using polling mode, you won't typically need to make any firewall changes on the Tentacle machine.
+
+### Intermediary firewalls
+
+Don't forget to allow access not just in Windows Firewall, but also any intermediary firewalls between the Tentacle and your Octopus Server. For example, if your Octopus Server is hosted in Amazon EC2, you'll also need to modify the AWS security group firewall to tell EC2 to allow the traffic. Similarly if your Octopus Server is hosted in Microsoft Azure you'll also need to add an Endpoint to tell Azure to allow the traffic.
+
+## Windows Server 2008 Limited Support
+From Octopus Server release `2024.1` there will be [limited support](https://octopus.com/docs/deprecations#dropping-capability-for-windows-server-2008-workers-and-targets-in-20251) for executing workloads on targets or workers running Windows Server 2008. As the version of [Calamari](https://octopus.com/docs/octopus-rest-api/calamari) shipped with later version of Octopus Server will no longer be compatable with Windows Server 2008, the capability has been made available to pin the version of Calamari used on the target itself. 
+
+To allow this:
+1. Download the [most recent version of Calamari](https://download.octopusdeploy.com/calamari/Calamari.Legacy.27.2.4-ci0003.zip) (currently `27.2.4-ci0003`). This is version of Calamari is unlikely to be updated unless there are significant vulnerbilities discovered so it may be missing new capabilities that are released later than `2025.1`.

```

Comment:
```suggestion
1. Download the [most recent version of Calamari](https://download.octopusdeploy.com/calamari/Calamari.Legacy.27.2.4-ci0003.zip) (currently `27.2.4-ci0003`). This is version of Calamari is unlikely to be updated unless there are significant vulnerabilities discovered so it may be missing new capabilities that are released later than `2025.1`.

```